setup.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python
 import os
+from pathlib import Path
```

Comment:
I take it that non-wheel installs would require Python 3? I think that's okay... We could make this nicer with a pyproject.toml though that imported pathlib2 for use in the setup.py on Python 2.